qipfs/config.go

```diff
@@ -26,6 +26,9 @@ type StoreCfg struct {
 	EnablePubSub bool
 	// DisableBootstrap will remove the bootstrap addrs from the node
 	DisableBootstrap bool
+	// ListeningAddrs allows you to add a list of addresses you want the
+	// underlying ipfs node to listen on
+	ListeningAddrs []string
```

Comment:
Rename this to `AdditionalSwarmListeningAddrs` and clarify that these are _in addition to_ addresses specified by IPFS for swarm listening. Any configuration specified as a filesystem configuration option needs to be clear about how it interacts with the IPFS config